tests/conftest.py

```diff
@@ -309,3 +321,10 @@ def __init__(
         endpoints_path: Text = "",
     ) -> None:
         super().__init__(tracker_store, event_broker, endpoints_path)
+
+
+class AsyncMock(Mock):
```

Comment:
https://docs.python.org/3.8/library/unittest.mock.html#unittest.mock.AsyncMock 😁 
I guess we have to support pre-3.8 though?

Reply:
Exactly 😬